README.md

```diff
@@ -1,2 +1,158 @@
+The detailed meaning of the fields is the following.
+- `request_id`: The value of the `request_id` field of the JSON object returned by an invocation of `get_batch`.
+  This value is required to pair the submission with the corresponding request on the server.
+- `proof_data`: The binary data produced by the prover, encoded as base64 string.
+- `proving_time`: The number of milliseconds it took to generate the proof. This **excludes** the time required for
+  downloading the batch. However, it must **include** the time for potentially allocating the required resources (if
+  that needs to be done for this proof), processing the batch, performing the actual proof generation, all the way until
+  the proof is ready to be submitted.
+- `cost`: The direct monetary cost of computing the proof US Dollar **cents**. It includes all the costs that are
+  incurred **per proof**, but not constant one-time investments necessary to start the proving.
+  
+  The `cost` **includes** items such as
+  - Rental cost for cloud machines (if cloud is used)
+  - Electricity to power the prover machines
+  - Cooling
+  - Network communication (e.g. ISP fees)
+  - Rent for the space where the machines are located
+  - Maintenance of the prover hardware and software
+  All these costs must be considered in proportion to what was actually used by the proving. For example, if the prover
+  hardware is co-located with other machines in the same room, only a fraction of the rent for that room and air
+  conditioning that corresponds to the consumption of the prover hardware must be considered.
+  
+  The `cost` **excludes** items such as
+  - Cost of proving hardware itself
+  - Engineers' compensation for setting up the proving system
+    (note that this is different from the engineer continuously maintaining the proving hardware and software,
+    which should flow into the proving cost)
+- `price`: The minimal price for which the participant is willing to perform the computation of the proof, including
+  the participant's profit margin. If this was an auction, the `price` would be the minimal value the participant would
+  be willing to bid.
+- `deployment_version`: Identifier of the deployment that produced the proof. This can be an arbitrary number or string
+  that identifies a particular deployment of hardware and software that has produced the submitted proof. If the
+  participant uses different deployments for proof computation (e.g. GPU-accelerated one and a CPU-only one, or
+  deployments with different numbers of machines), this field should be used to distinguish them. The value of this
+  field need not fully describe the deployments. It can merely serve as a reference to a description of the deployment
+  stored elsewhere and communicated separately to the experiment operator.
+
+
+> **Note**: In the current setup, where batches are proven one by one, the request ID is not strictly necessary, as the
```

Comment:
Let's remove this note. If participants don't submit the request ID, it could make data processing messier